### ISSUE TITLE
Security vulnerability when installing Onsen UI

### ISSUE THREAD

**Issue by puku0x** [OnsenUI/OnsenUI#2589]
__Environment__

Onsen UI Version:

- 2.10.5

__Encountered problem__

 I got a security vulnerability after I installed Onsen UI via `npm install onsenui` because  it had `gulp-protractor` as a dependency.

```
$ npm ls event-stream flatmap-stream
my-app@0.0.0
└─┬ onsenui@2.10.5
  └─┬ gulp-protractor@4.1.0
    └── event-stream@3.3.5
```

It should be in `devDependencies`, right?

**Comment by masahirotanaka** [comment]
Right. We will fix this and release by today or tomorrow.

**Comment by puku0x** [comment]
@masahirotanaka Thank you for your reply! I hope it will be fixed soon🙏

**Comment by masahirotanaka** [comment]
Looks like `gulp-protractor` has updated to 4.1.1, so this issue should has been solved now.
I'm asking @emccorson to double check.

**Comment by puku0x** [comment]
@masahirotanaka Thanks for the info. I'll reinstall Onsen UI later.
BTW, why the `gulp-protractor` was in the `dependencies: {}` in `package.json`? I think it's not required for the end-users.

**Comment by emccorson** [comment]
@puku0x Thanks for reporting this. I had read about this problem but didn't realise it affected Onsen UI  so that's a good catch!

It looks like the warnings were because of the event-stream vulnerability (dominictarr/event-stream#116). I've changed the package version of `gulp-protractor` to `^4.1.1` and the version of `event-stream` to `3.3.4` just to be on the safe side (#2590).

Still looking into whether `gulp-protractor` should be a dependency. I think it was to do with splitting the AngularJS bindings into their own package. I'll comment again when I find out.

**Comment by puku0x** [comment]
@emccorson Thank you for telling me about the details. I'm looking for the next release.

**Comment by emccorson** [comment]
Should be fixed now in v2.10.6.
